resteasy-upgrade-guide/reference/en/en-US/modules/Miscellaneous.xml

```diff
@@ -121,4 +121,49 @@ ResteasyProviderFactory.getInstance().registerProvider(POJOConverterProvider.cla
    </para>
    
    </section>
+
+   <section>
+      <title>Matching Requests to Resource Methods</title>
+
+      <para>
+         Improvements and corrections have been made in Resteasy 3 to the implementation of
+         the matching rules as defined in the JAX-RS 2.0 specification.  In particular
+         a change for handling an ambiguous uri on a sub-resource method and
+         sub-resource locator was made.  In Resteasy 2 it was possible for a
+         sub-resource locator to execute successfully even when there was also a sub-resource
+         with the same uri.  This was incorrect behavior according to the specification.
+         In Resteasy 3 when there is an ambiguous uri for a sub-resource and a
+         sub-resource locator.  Calling the sub-resource will be successful.  Calling
+         the sub-resource locator will result in an HTTP, 405 Method Not Allowed error.

```

Comment:
@rsearls, thanks for the PR. I'm probably too picky, but can you fix the punctuation here (the 3 parts of the sentence here should probably be together) ?